### PR TITLE
Added icons to navigation drawer #47

### PR DESCRIPTION
**Pull request by NoodleMage**
Added icons to navigation drawer. Moved settings to the bottom of nav drawer.

### CODE REVIEW

**Comment by NoodleMage** [comment]
screenshot_2016-01-27-12-58-38

**Comment by inorichi** [comment]
I don't particularly like the settings at the bottom. If you open the drawer from the top, you are forced to move the hand to open the settings on a tablet.

Any other opinion on this?

**Comment by Rumple007** [comment]
That format looks pretty good and usually people dont access settings a lot except twice or thrice to set up  the app as you want so I dont think it would make things difficult

**Comment by NoodleMage** [comment]
As by Android guidelines:
https://www.google.com/design/spec/patterns/navigation-drawer.html#navigation-drawer-content

Settings and support

Settings and support are located at the bottom of the scrolling list, inline with the rest of the list content. They refer to Help, Feedback, or Help & feedback, depending on what your product offers.

That's why I put it at the bottom :)

**Comment by inorichi** [comment]
"inline with the rest of the list" doesn't mean it should be below the others? I'm not sure but I think it means that.

**Comment by NoodleMage** [comment]
I see it done like this in almost every application I use (that uses material design). So if you want I can put it below download queue. But in my opinion this looks cleaner

**Comment by NoodleMage** [comment]
Just discovered I have been wrong my entire life :o.

Updated:
screenshot_2016-01-27-15-10-16

**Comment by inorichi** [comment]
Yup, that's what I was meaning :dancer:

**Comment by M2ck** [comment]
Perfect :+1:

**Comment by Bloodwork** [comment]
Looks great!

File: app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.java [hunk 1]
```diff
@@ -26,15 +28,17 @@
 
 public class MainActivity extends BaseActivity {
 
-    @Bind(R.id.appbar) AppBarLayout appBar;
-    @Bind(R.id.toolbar) Toolbar toolbar;
-    @Bind(R.id.drawer_container) FrameLayout container;
-
+    @Bind(R.id.appbar)
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.java]: Everything in a single line please :D

**Pull request merged**